internal/database/migrations/mysql/0001_initial.up.sql

```diff
@@ -1,8 +1,9 @@
 CREATE TABLE IF NOT EXISTS account(
-		id       INT(11)      NOT NULL AUTO_INCREMENT,
-		username VARCHAR(250) NOT NULL,
-		password BINARY(80)   NOT NULL,
-		owner    TINYINT(1)   NOT NULL DEFAULT '0',
+		id       INT(11)        NOT NULL AUTO_INCREMENT,
+		username VARCHAR(250)   NOT NULL,
+		password BINARY(80)     NOT NULL,
+		owner    TINYINT(1)     NOT NULL DEFAULT '0',
+		configures VARCHAR(500) NOT NULL,
```

Comment:
I'd go with `config` or `configuration` as the name. And I'm not sure if you are using this as test, but the table alteration should be on a new migration (a new file). You need to use `ALTER TABLE` to modify the table, to allow current shiori deployments to upgrade seamlessly to this new schema.

Just add a new file adding 1 to the number of the previous one (if the last one is `0004`, you create `0005` and so on), and put the `ALTER TABLE` inside, it should work automatically if the SQL is right.

Reply:
Also, I think we can use `JSON` in all engines, I'm creating a snippet for you to check, because we should be able to use sqlx to deserialize directly to a struct.

Reply:
Here's a playground: https://go.dev/play/p/QDlGfg2TMYZ

It will probably timeout due to the dependency download, but it should work locally so you can get an idea. it should be even easier for psql and mariadb I guess.